script.js

```diff
@@ -126,34 +192,36 @@ function makePageForEpisodes(episodeList) {
 
     containerElement.appendChild(episodeContainer);
   });
+  const searchContainer = getElementById("search-container");
+  searchContainer.innerHTML = "";
+  searchContainer = searchFunction(episodeList);
 }
 
-
-
 function makeFooter(allEpisodes) {
   const footer = document.createElement("footer");
   footer.className = "footer-container";
-  footer.innerHTML = 
-    `Data Collected from TVMaze.com
+  footer.innerHTML = `Data Collected from TVMaze.com
     <br>
     Found ${allEpisodes.length} episode(s)`;
   document.body.appendChild(footer);
 }
 
+async function initialise() {
```

Comment:
It's great that you have separated each part of the page into its own function. Looks clean and makes it easy to read the code. Thanks!